gpAux/gpdemo/README

```diff
@@ -33,7 +33,7 @@ RUNNING GP DEMO
 
 2. Source greenplum_path.sh
 
-	. /usr/local/greenplum-db/greenplum_path.sh
+	. /usr/local/gpdb/greenplum_path.sh
```

Comment:
Should we change here?

Reply:
> The configure default prefix is `/usr/local/gpdb`, and the top
README also use `/usr/local/gpdb` as the prefix, so there is
large chance that developer will use `/usr/local/gpdb` instead
of `/usr/local/greenplum_db`.
This change will eliminate the newcomers' confusion.

make sense.